quickstarts/calico/config.yml

```
@@ -0,0 +1,48 @@
+id:
+slug: calico newrelic
+title: Calico Newrelic
+
+description: |
+
+  Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico supports a broad range of platforms including Kubernetes, OpenShift, Mirantis Kubernetes Engine (MKE), OpenStack, and bare metal services.
+
+Whether you opt to use Calico's eBPF data plane or Linux’s standard networking pipeline, Calico delivers blazing fast performance with true cloud-native scalability
+
+  Install this quickstart to access preconfigured Calico observability solutions based on Prometheus metrics that are available out-of-the-box
+
+summary: |
+
+   A curated dashboard based on Prometheus metrics which helps you understand the Calico CNI of your k8s cluster
+
+# Support level: New Relic | Verified | Community (required)
+level:  Community
```

Looks like a extra space here
```suggestion
level: Community
```

---

Corrected